.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
```

Comment:
Why do you exclude this dist folder, because it can contain a packaged version of this parse module?